### PR TITLE
add footer functionality

### PR DESCRIPTION
**Pull request by deoxxa**

### CODE REVIEW

File: table.go [hunk 1]
```diff
@@ -57,10 +71,12 @@ func (t *Table) AddRow(row map[string]interface{}) {
 
 // Print - Prints table.
 func (t *Table) Print() {
-	if len(t.Rows) == 0 {
+	if len(t.Rows) == 0 && t.Footer == nil {
```
Comment (crackcomm) [inline table.go]: So no Table with no Footer?

**Comment by crackcomm** [comment]
Sorry for the delay but anyway the footer functionality looks like just another row so Im wouldn't merge it anyway. Thanks for the try anyway.

**Comment by deoxxa** [comment]
It's actually different to a regular row. It has a separator before it, just like the header has after it.

**Comment by crackcomm** [comment]
Now I reviewed my thinking It could be actually useful, for printing sums at the bottom etc. If you want to create a new pull request I will merge this.

**Comment by deoxxa** [comment]
`git pull git://github.com/deoxxa/go-clitable.git add-footer:master` should work easily enough.

**Comment by crackcomm** [comment]
Here is a markdown table with a footer:

```
| Name | Cost  |
| ---- | ----- |
| test | 1.19$ |
| test | 0.10$ |
| ---- | ----- |
|      | 1.29$ |
```

Here is a non-markdown table.

```
|--------------|
| Name | Cost  |
|--------------|
| test | 1.19$ |
|--------------|
| test | 0.10$ |
|--------------|
|--------------|
|      | 1.29$ |
```

I modified it a little bit to make it a closed table and it looks like this:

```
|--------------|
| Name | Cost  |
|--------------|
| test | 1.19$ |
|--------------|
| test | 0.10$ |
|--------------|
|--------------|
|      | 1.29$ |
|--------------|
```

**Comment by deoxxa** [comment]
Looks great!

**Pull request closed without merging**